packages/patternfly-4/react-core/src/components/Dropdown/DropdownToggleAction.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/Dropdown/dropdown';
+import { css } from '@patternfly/react-styles';
+
+export interface DropdownToggleActionProps {
+  /** Additional classes added to the DropdownToggleAction */
+  className?: string;
+  /** Flag to show if the action button is disabled */
+  isDisabled?: boolean;
+  /** A callback for when the action button is clicked */
+  onClick?(event: React.MouseEvent<HTMLButtonElement>): void;
+  /** Element to be rendered inside the <button> */
+  children?: React.ReactNode;
+  /** Id of the action button */
+  id: string;
```

Comment:
@tlabaj left a comment here but looks like it was removed. The `id` shouldn't be required. `id` is required on the button with the arrow (the one that toggles the menu), but for a normal action there is no requirement for an `id`.